lib/src/screens/explore/community_mod_panel.dart

```diff
@@ -165,3 +187,324 @@ class _CommunityModPanelBansState extends State<CommunityModPanelBans> {
     super.dispose();
   }
 }
+
+class CommunityModPanelReports extends StatefulWidget {
```

Comment:
I just have a few suggestions for the UI here, as I'd say things seem a bit cluttered/verbose at the moment.

- Can the View button be removed and replaced with an InkWell that opens the reported content when the report is clicked on?
- Can the Ban User button be changed to include the actual user's username, similar to how Mbin does it? Then from here, we can also remove the "Reported user" text, as you can see which user is being reported in the Ban X button.
- Maybe to simplify the actions, hide the reject button if the report is already rejected, hide the accept button if already approved, and display both if still pending; that's what Mbin does for reports as well.

Let's see how it looks after your next changes and go from there.